src/store/getters.ts

```diff
@@ -265,14 +331,26 @@ export default {
     const { state } = rootGetterContext(context);
     const { fiatRates } = state;
     return (asset: AssetType, balance: BigNumber): BigNumber | null => {
-      if (fiatRates && fiatRates[asset] && balance) {
+      if (fiatRates && fiatRates[asset] && balance.gt(0)) {
         const amount = unitToCurrency(cryptoassets[asset], balance);
         // TODO: coinformatter types are messed up and this shouldn't require `as BigNumber`
         return cryptoToFiat(amount, fiatRates[asset]) as BigNumber;
       }
       return null;
     };
   },
+  assetMarketCap(...context: GetterContext) {
+    const { state } = rootGetterContext(context);
+    const { currenciesInfo } = state;
+
+    return (asset: AssetType): BigNumber | null => {
```

Comment:
you can use `Nullable` type from Chainify

Reply:
applied